package.json

```diff
@@ -173,6 +173,8 @@
         "@react-types/dialog": "3.4.5",
         "@react-types/shared": "3.16.0",
         "@tailwindcss/forms": "^0.5.3",
+        "@typescript-eslint/eslint-plugin": "^5.59.1",
```

Comment:
Update the Frontify config instead, it is the reason

Reply:
Thanks, Right! I just noticed this Pr: https://github.com/Frontify/fondue/pull/1367/files
There are some linting issue to take care of as well. Taking a look in a bit.